src/NpmUtilities.js

```diff
@@ -101,55 +101,54 @@ export default class NpmUtilities {
   }
 
 
-  static addDistTag(directory, packageName, version, tag, registry) {
+  static addDistTag(directory, packageName, version, tag, {client = 'npm', registry} = {}) {
     log.silly("addDistTag", tag, version, packageName);
-
     const opts = NpmUtilities.getExecOpts(directory, registry);
-    ChildProcessUtilities.execSync("npm", ["dist-tag", "add", `${packageName}@${version}`, tag], opts);
+    ChildProcessUtilities.execSync(client, ["dist-tag", "add", `${packageName}@${version}`, tag], opts);
```

Comment:
There's actually no `yarn dist-tag` command. I don't see any reason to change `addDistTag`/`removeDistTag`/`checkDistTag`.

Reply:
Ok, I decided to remove them, instead of leaving it there and always calling with `npm`, I guess its less confusing..